### DISCUSSION TITLE
[Discord Server] NESTang Discord Server

### DISCUSSION THREAD

**Discussion by fjpolo** [nand2mario/nestang#46] [General]
Created a Discord Server: [NESTang @Discord](https://discord.gg/zUM9fQ86)

🚧 Under construction🚧

**Comment by nikitalita** [comment]
@fjpolo can you update the link? The invite has expired

**Comment by fjpolo** [comment]
Newest @20240605: https://discord.gg/CJaZqXsp

**Comment by fjpolo** [comment]
https://discord.gg/uPjcdjnc

**Reply by Roet-Ivar** [reply]
Is there any new link? 😊
